### ISSUE TITLE
Read Netflow packets from a pcap file

### ISSUE THREAD

**Issue by jakane** [phaag/nfdump#165]
We have a process that saves netflow packets on our network into pcap files (we send netflow to a specific collector on UDP/35000)

I need to be able to convert that PCAP file (with flow packets in it) into NFCAP files for further analysis.

I have looked at nfpcapd and that does not appear to do what I want, it essentially acts like a netflow exporter as best as I can tell. My PCAP has the flow packets already, I just want something like:

nfcapd < pcap_file

Any hints?

Thanks

**Comment by phaag** [comment]
There is an option in nfcapd in order to read your pcaps, but it was intended for development and debugging purpose. If you configure the package with ./configure --enable-readpcap, you enable this debugging code in nfcpad. To read now the pcap you need to start nfcapd as you would start it otherwise, but instead of a port to listen at (-p), you specify the pcap file with -f <pcapfile>

**Comment by jakane** [comment]
Respectfully, I read the docs and am aware of the option. What the option does is take captured pcap files and then acts like a netflow engine and outputs resulting new netflow packets. I have tested it and it works as advertised.

What I am asking for is not the same.

We have a very large, home-grown netflow collection system. We have distributed flow collectors that are "close" to our routers where we collect the flow packets and do some processing of them, storing the enriched data locally on each collector. That stored data is then transported reliably to a central collection system for analysis.

We now have a component in the system that requires the flow packets to be delivered directly to it. Our system already supports mirroring packets to additional locations and the mirroring works for this additional system. What we do not have is any way to recover from an outage at the new centralized system. Our thought is to capture netflow packets at each distributed collector and then replay those packets to the central server when it is no longer impaired. It felt like a viable use case for nfcapd, but if you think that we should just restrict our needs to something like `tcpreplay` then we can accept that.
